pkg/sidecar/Dockerfile

```diff
@@ -0,0 +1,43 @@
+#:::
+#::: BUILD CONTAINER
+#:::
+
+# GO_VERSION is the golang version this image will be built against.
+ARG GO_VERSION
+
+# Dynamically select the golang version.
+# TODO: Not sure how this interplays with image caching.
+FROM golang:${GO_VERSION}-buster
+
+# GO_VERSION is the golang version this image will be built against.
+ARG GO_VERSION
+
+# Unfortunately there's no way to specify a ** glob pattern to cover all go.mods
+# inside sdk.
+COPY /sdk/sync/go.mod /sdk/sync/go.mod
+COPY /sdk/runtime/go.mod /sdk/runtime/go.mod
+COPY /go.mod /go.mod
+
+# Download deps.
+RUN cd ${PLAN_DIR} \
+    && go env -w GOPROXY=direct \
```

Comment:
I think we can safely remove the GOPROXY=direct here, and allow it to use Google's all-watching default proxy...